src/openrct2/Game.cpp

```diff
@@ -279,29 +296,31 @@ void update_palette_effects()
             }
         }
 
-        j = ((uint16)(gPaletteEffectFrame * -960) * 3) >> 16;
+        j       = ((uint16) (gPaletteEffectFrame * -960) * 3) >> 16;
         waterId = SPR_GAME_PALETTE_4;
-        g1 = gfx_get_g1_element(shade + waterId);
-        if (g1 != NULL)
+        g1      = gfx_get_g1_element(shade + waterId);
```

Comment:
This doesn't look aligned properly. Your formatting tool commonly mistakes assignments with other uses of `=`, please keep an eye out for that.

Reply:
This is aligned properly though. It aligns with `waterId = SPR_GAME_PALETTE_4;`:
```
        j       = ((uint16) (gPaletteEffectFrame * -960) * 3) >> 16;
        waterId = SPR_GAME_PALETTE_4;
        g1      = gfx_get_g1_element(shade + waterId);
        if (g1 != nullptr)
```
I think you're thrown off by the diff.